docs/content/index.html

```diff
@@ -15,7 +15,35 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width">
-
+	<link rel="schema.dcterms" href="http://purl.org/dc/terms/">
+	<link rel="schema.AGLSterms" href="http://www.agls.gov.au/agls/terms/">
+	<meta name="dcterms.title" content="Content page template - Australian Government Design System" />
+	<meta name="dcterms.creator" content="scheme=AGLSTERMS.AglsAgent; corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=+61 427 136 791" />
+	<meta name="dcterms.publisher" content="scheme=AGLSTERMS.AglsAgent; corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=+61 427 136 791" />
+	<meta name="dcterms.description" content="Content pages display information clearly" />
+	<meta name="dcterms.modified" content="scheme=dcterms.ISO8601; 2019-08-06T11:36:29+10:00" />
+	<meta name="dcterms.subject" content="design system, templates" />	
+	<meta name="dcterms.language" content="dcterms.RFC4646; en-AU" />
+	<meta name="dcterms.type" content="Content page template" />
+	<meta name="dcterms.coverage" content="Australia" />
+	<meta name="dcterms.spatial" content="Australia" />
+	<meta name="AGLSTERMS.documentType" content="text" />
+	<meta name="msapplication-TileColor" content="#ffffff">
```

Comment:
Suggest inclusion of `identifier`, reusing value in `og:url`: 

`<meta name="DCTERMS.identifier" scheme="DCTERMS.URI" content="https://www.designsystem.gov.au/path/to/page/" />`
 